opennebula/defaults.yaml

```diff
@@ -2,50 +2,42 @@ kernel:
   Linux:
     oneadmin:
       home: /var/lib/one
-      optional_groups:
-        - kvm
-        - libvirt
-        - ssl-cert
       config:
         manage:
-          - ec2_auth
-          - occi_auth
+#          - ec2_auth
```

Comment:
Yeah..., I' ve seen and tried this, settings in the pillar override default settings:
so doing a:
opennebula:
  lookup:
    oneadmin:
      config:
        manage: []

prevents configuring this files at all.
But still I'm a bit scared of changing the auth files on every salt run.
For one_auth and one_key I would like to see something like this
file.managed:
  ...
  - unless test -f /var/lib/one/.one/one_auth
  ...
because simply changing the file without changing the database does not work.
I will implement a patch where I create those 2 files if they don't exist.
I will send you a pull request for this and we can discuss again if you like it :)